classification/templates/classification/classification_groups_mvl.html

```diff
@@ -22,11 +23,16 @@
                     <del>{{ group.clinical_significance | ekey:'clinical_significance' }}</del><br/>{{ group.clinical_significance_pending | ekey:'clinical_significance' }} (PENDING)
                 {% else %}{{ group.clinical_significance | ekey:'clinical_significance' }}{% endif %}
             </td>
-            <td style='padding:8px'>{% for condition in group.conditions %}<div>{% condition condition_obj=condition limit=None %}</div>{% empty %}-{% endfor %}</td>
+            <td style='padding:8px'>{% for condition in group.conditions %}<div>{% condition condition_obj=condition limit=None no_links=True %}</div>{% empty %}-{% endfor %}
```

Comment:
OntologyTerm has "show_link" which defaults to True, probably best to keep the convention (instead of "no_links" defaulting to False).